src/screens/WaitingScreen/index.js

```diff
@@ -0,0 +1,19 @@
+import './styles.css'
+
+import { useDoc } from "@syncstate/react";
+
+const WaitingScreen = () => {
+  const [activeUser] = useDoc("/activeUser");
+  const notEnoughPlayers = activeUser < 2;
+  const toManyPlayers = activeUser > 2;
+
+  return (
+    <div>
+      {notEnoughPlayers && <h2>Waiting for an other player ...</h2>}
+      {toManyPlayers && <h2>To many players ...</h2>}
```

Comment:
```suggestion
      {tooManyPlayers && <h2>Too many players ...</h2>}
```

Not sure about this one. Couldn't you prevent another player to join when 2 are already playing?

Reply:
Don't know what actually happens here, but maybe provide some information to the 3rd player that he can't join but he could watch? 🤯